src/Sylius/Bundle/UiBundle/ContextProvider/ContextProviderInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\UiBundle\ContextProvider;
+
+use Sylius\Bundle\UiBundle\Registry\TemplateBlock;
+
+/**
+ * @experimental
+ */
```

Comment:
```suggestion
/** @experimental */
```

Reply:
I forgot to refactor it but in the next PR, I will completely remove `@experimental` tags so I would be for leaving it as it is for now.

Reply:
And all of them, I guess, are written with 3 lines so it will be easier to do search&replace :grinning: